integrationTesting/tests/organize/surveys/submitting-survey.spec.ts

```diff
@@ -0,0 +1,191 @@
+    );
+  });
+
+  test.afterEach(({ moxy }) => {
+    moxy.teardown();
+  });
+
+  test('submits responses', async ({ moxy, page }) => {
+    moxy.setZetkinApiMock(
+      `/orgs/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}/submissions`,
+      'post',
+      {
+        timestamp: '1857-05-07T13:37:00.000Z',
+      }
+    );
+
+    await page.click('input[name="1.options"]');
+    await page.fill('[name="2.text"]', 'Topple capitalism');
+    await page.click('input[name="sig"][value="anonymous"]');
+    await page.click('data-testid=Survey-acceptTerms');
+    await Promise.all([
+      page.waitForResponse((res) => res.request().method() == 'POST'),
+      await page.click('data-testid=Survey-submit'),
+    ]);
+
+    const log = moxy.log(`/v1${apiPostPath}`);
+    expect(log.length).toBe(1);
+    const [request] = log;
+    const data = request.data as {
+      responses: ZetkinSurveyQuestionResponse[];
+    };
+    expect(data.responses.length).toBe(2);
+    expect(data.responses).toMatchObject([
+      {
+        options: [1],
+        question_id: KPDMembershipSurvey.elements[0].id,
+      },
+      {
+        question_id: KPDMembershipSurvey.elements[1].id,
+        response: 'Topple capitalism',
+      },
+    ]);
+  });
+
+  test('submits email signature', async ({ moxy, page }) => {
+    moxy.setZetkinApiMock(
+      `/orgs/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}/submissions`,
+      'post',
+      {
+        timestamp: '1857-05-07T13:37:00.000Z',
+      }
+    );
+
+    await page.click('input[name="1.options"]');
+    await page.fill('[name="2.text"]', 'Topple capitalism');
+    await page.click('input[name="sig"][value="email"]');
+    await page.fill('input[name="sig.email"]', 'testuser@example.org');
+    await page.fill('input[name="sig.first_name"]', 'Test');
+    await page.fill('input[name="sig.last_name"]', 'User');
+    await page.click('data-testid=Survey-acceptTerms');
+    await Promise.all([
+      page.waitForResponse((res) => res.request().method() == 'POST'),
+      await page.click('data-testid=Survey-submit'),
+    ]);
+
+    const log = moxy.log(`/v1${apiPostPath}`);
+    expect(log.length).toBe(1);
+    const [request] = log;
+    const data = request.data as {
+      signature: ZetkinSurveySignaturePayload;
+    };
+    expect(data.signature).toMatchObject({
+      email: 'testuser@example.org',
+      first_name: 'Test',
+      last_name: 'User',
+    });
+  });
+
+  test.skip('submits user signature', async ({ moxy, page }) => {
+    moxy.setZetkinApiMock(
+      `/orgs/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}/submissions`,
+      'post',
+      {
+        timestamp: '1857-05-07T13:37:00.000Z',
+      }
+    );
+
+    await page.click('input[name="1.options"][value="1"]');
+    await page.fill('[name="2.text"]', 'Topple capitalism');
+    await page.click('input[name="sig"][value="user"]');
```

Comment:
I see this working in the test (which is passing) but for some reason I'm not seeing this input when I'm using the page while authenticated. Lets have a look together today!

Reply:
This test is actually skipped!

```
test.skip('submits user signature', async ({ moxy, page }) => {
```